R/predict.R

```diff
@@ -59,36 +56,41 @@
 #' produces. Set `increasing = FALSE` to suppress this behavior.
 #'
 #' @return With the exception of `type = "raw"`, the results of
```

Comment:
```suggestion
#' @return With the exception of `type = "raw"`, the result of
```

otherwise reads "the results of `predict.model_fit()` is a tibble," etc.